parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -355,8 +361,10 @@ async def folder_create(self, path: FsPath) -> EntryID:
             await self.local_storage.set_manifest(parent.id, new_parent)
 
         # Send events
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent.id)
-        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=child.id)
+        self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent.id, reason=None)
+        self._send_event(
+            CoreEvent.FS_ENTRY_UPDATED, id=child.id, reason=FSEntryUpdatedReason.FOLDER_CREATE
```

Comment:
Here we can use another reason like `FSEntryUpdatedReason.FOLDER_CREATE_ENTRY_CREATION` (and it would be good to have a comment explaining the difference between `FOLDER_CREATE` and `FOLDER_CREATE_ENTRY_CREATION` in `FSEntryUpdatedReason` definition)